colosseum/declaration.py

```diff
@@ -43,16 +48,26 @@ def deleter(self):
     return property(getter, setter, deleter)
 
 
-def validated_property(name, choices, initial):
+def validated_property(name, choices, initial, storage_class=None):
     "Define a simple CSS property attribute."
-    initial = choices.validate(initial)
+
+    def wrap_storage_class(value):
```

Comment:
I can see why you're doing this, but it feels like the wrong approach. This is glossing over a parsing behavior - which suggests to me that what we need here is a parser that returns the appropriate object based on the input. The default parser is "return the value"; but border_spacing can add a parser that filters out INHERIT values, and validates the various ways of expressing border spacing.

Reply:
Fixed!